docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -59,13 +63,51 @@ To set up the AI Assistant:
 [[obs-ai-add-data]]
 == Add data to the AI Assistant knowledge base
 
+[IMPORTANT]
+====
+*If you started using the AI Assistant in technical preview*,
+any knowledge base articles you created before 8.12 will no longer be supported.
+Knowledge base articles created before 8.12 use ELSER v1.
+In 8.12, knowledge base articles must use ELSER v2.
+You can either:
+
+* Clear all old knowledge base articles manually and reindex them.
+* Upgrade all knowledge base articles indexed with ELSER v1 to ELSER v2 using a https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/model-upgrades/upgrading-index-to-use-elser.ipynb[Python script].
+====
+
 The AI Assistant uses {ml-docs}/ml-nlp-elser.html[ELSER], Elastic's semantic search engine, to recall data from its internal knowledge base index to create retrieval augmented generation (RAG) responses. Adding data such as Runbooks, GitHub issues, internal documentation, and Slack messages to the knowledge base gives the AI Assistant context to provide more specific assistance.
 
 NOTE: Your AI provider may collect telemetry when using the AI Assistant. Contact your AI provider for information on how data is collected.
 
 You can add information to the knowledge base by asking the AI Assistant to remember something while chatting (for example, "remember this for next time"). The assistant will create a summary of the information and add it to the knowledge base.
 
-You can also add external data to the knowledge base by completing the following steps:
+You can also add external data to the knowledge base either in {kib} using the Stack Management UI or using the {es} Index API.
```

Comment:
Should we include a URL to the specific stack management page?

Also we using Serverless the Stack Management UI is not available, but many of its features including the Ai assistant ones are available in the Project Settings UI. Should we include this here?

Reply:
>Should we include a URL to the specific stack management page?

I don't think we can add a URL because it's specific to each deployment, right? 

>Also we using Serverless the Stack Management UI is not available, but many of its features including the Ai assistant ones are available in the Project Settings UI. Should we include this here?

I'll open a separate PR to update the serverless docs once this PR is finalized! I'll be sure to update the language that describes to the UI to reflect what exists in serverless.

Reply:
The base for the URL will be different for every deployment, but the last part will be the same.

Stack management: 
* app/management

AI Assistant Management:
* app/management/kibana/aiAssistantManagementSelection

AI assistant for Observability
* app/management/kibana/aiAssistantManagementObservability

Reply:
>Should we include a URL to the specific stack management page

My two cents: As far as I know, we don't typically document the URL. Instead we document the navigation steps. Because URL changes are subtle, we are less likely to notice them. When the navigation steps change, we'll do global updates to fix the problem.